css/utils/mediaQueries.css

```diff
@@ -49,8 +49,9 @@
     align-items: center;
   }
 
-  .cart-container {
-    padding-top: 10rem !important;
+  .cart-container,
+  .wishlist-container {
+    padding-top: 11rem !important;
```

Comment:
Can't important be avoided here? If possible try to use specificity

Reply:
Updated